### PR TITLE
Initial writeup with references

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+## Interfaces 
+* **Pharus** -- a REST API for interacting with DataJoint databases  https://github.com/datajoint/pharus
+* **DataJoint Labbook** -- a front-end web interface for viewing and entering data  https://github.com/datajoint/datajoint-labbook
+
+## Elements 
+
+* Lab management https://github.com/datajoint/elements-lab 
+* Animal  management https://github.com/datajoint/elements-lab 
+* Extracellular electrophysiology https://github.com/datajoint/elements-ephys
+* Calcium imaging https://github.com/datajoint/delements-imaging
```
Comment (ttngu207) [inline README.md:60]:
```suggestion
* Calcium imaging https://github.com/datajoint/elements-imaging
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+* Interactive online tutorial https://playground.datajoint.io
+
+## Interfaces 
+* **Pharus** -- a REST API for interacting with DataJoint databases  https://github.com/datajoint/pharus
+* **DataJoint Labbook** -- a front-end web interface for viewing and entering data  https://github.com/datajoint/datajoint-labbook
+
+## Elements 
+
+* Lab management https://github.com/datajoint/elements-lab 
+* Animal  management https://github.com/datajoint/elements-lab
```
Comment (ttngu207) [inline README.md:58]:
```suggestion
* Animal management https://github.com/datajoint/elements-animal
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+
+* Lab management https://github.com/datajoint/elements-lab 
+* Animal  management https://github.com/datajoint/elements-lab 
+* Extracellular electrophysiology https://github.com/datajoint/elements-ephys
+* Calcium imaging https://github.com/datajoint/delements-imaging 
+
+## Example workflows 
+
+* Electrophysiology workflow with lab and animal management: https://github.com/datajoint/workflow-ephys
+* Imaging workflow https://github.com/datajoint/workflow-imaging
```
Comment (kabilar) [inline README.md:65]:
```suggestion
* Imaging workflow with lab and animal management https://github.com/datajoint/workflow-imaging
```

**Review by kabilar — APPROVED** [review]
Great work.  Should we add a section to discuss the NIH U24 funding?

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**